cmd/kubeadm/app/apis/kubeadm/timeoututils.go

```diff
@@ -32,6 +32,7 @@ func SetDefaultTimeouts(t **Timeouts) {
 		EtcdAPICall:                      &metav1.Duration{Duration: constants.EtcdAPICallTimeout},
 		TLSBootstrap:                     &metav1.Duration{Duration: constants.TLSBootstrapTimeout},
 		Discovery:                        &metav1.Duration{Duration: constants.DiscoveryTimeout},
+		UpgradeManifests:                 &metav1.Duration{Duration: constants.UpgradeManifestsTimeout},
```

Comment:
i could not find anything else that is worth adding. if you find something LMK.

well...there are two more things that can be added, but they might be a bit tricky because they are used very early in the command execution.

- our global kubeconfig -> client timeout is 10s:
https://github.com/kubernetes/kubernetes/blob/d8122df1b73d0d92ebc32a7be9d8128d33f13dab/cmd/kubeadm/app/util/kubeconfig/kubeconfig.go#L83

- our "get k8s version" from label is also hardcoded:

https://github.com/kubernetes/kubernetes/blob/d8122df1b73d0d92ebc32a7be9d8128d33f13dab/cmd/kubeadm/app/util/version.go#L38